dialogflow_v2beta1/proto/knowledge_base_pb2.py

```diff
@@ -83,6 +83,24 @@
             serialized_options=None,
             file=DESCRIPTOR,
         ),
+        _descriptor.FieldDescriptor(
+            name="language_code",
+            full_name="google.cloud.dialogflow.v2beta1.KnowledgeBase.language_code",
+            index=2,
+            number=4,
+            type=9,
+            cpp_type=9,
+            label=1,
+            has_default_value=False,
+            default_value=_b("").decode("utf-8"),
+            message_type=None,
+            enum_type=None,
+            containing_type=None,
+            is_extension=False,
+            extension_scope=None,
+            serialized_options=None,
+            file=DESCRIPTOR,
+        ),
```

Comment:
@busunkim96, 

To add to the confusion, this isn't the problematic `language_code` field, but if this has no default value, won't this also be a required field and since it is new that would be a breaking change or am I reading this wrong?

Reply:
Are you allowed to pass an empty language_code to the API? If that's true it might be alright that there's no default value here.